charts/ps-operator/templates/role-binding.yaml

```diff
@@ -44,3 +52,25 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: {{ include "ps-operator.fullname" . }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ include "ps-operator.fullname" . }}-orchestrator
+  {{- if .Values.watchNamespace }}
+  namespace: {{ .Values.watchNamespace }}
+  {{- end }}
+  labels:
+{{ include "ps-operator.labels" . | indent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  {{- if .Values.watchNamespace }}
+  kind: ClusterRole
+  {{- else }}
+  kind: Role
+  {{- end }}
+  name: {{ include "ps-operator.fullname" . }}-orchestrator
+subjects:
+- kind: ServiceAccount
+  name: {{ include "ps-operator.fullname" . }}-orchestrator
```

Comment:
Is it ok to keep auto generated service account name reference here when `serviceAccount.create=false`?

Reply:
You are right, it might be problematic, but we have a general issue with orchestrator service account so we will need changes here later anyway, I have opened a ticket here: https://jira.percona.com/browse/K8SPS-235